Documentation/extending-form-api.rst

```diff
@@ -177,9 +178,13 @@ list. A Form Element Renderer must implement the ``RendererInterface`` interface
 	         $content .= sprintf('<li>%s</li>', htmlspecialchars($item));
 	      }
 	      $content .= '</ul>';
+	      $content = $this->formRuntime->invokeRenderCallbacks($content, $renderable);
 	      return $content;
 	   }
 	}
 
+.. note::  Don't forget to invoke ``RootRenderableInterface::beforeRendering()`` and ``FormRuntime::invokeRenderCallbacks()``
```

Comment:
Nah, it's part of the API, so AOP looks too much tacked-on. But maybe signal-slot could be used? Or just do it like suggested…